README.md

```diff
@@ -53,6 +53,15 @@ There's a multitude of settings you can use to control the plugin server. Use th
 | CELERY_DEFAULT_QUEUE          | Celery outgoing queue                                             | `'celery'`                            |
 | PLUGINS_CELERY_QUEUE          | Celery incoming queue                                             | `'posthog-plugins'`                   |
 | PLUGINS_RELOAD_PUBSUB_CHANNEL | Redis channel for reload events                                   | `'reload-plugins'`                    |
+| POSTHOG_DB_NAME               | Postgres database name                                            | `null`                                |
+| POSTHOG_DB_USER               | Postgres database user                                            | `'postgres'`                          |
+| POSTHOG_DB_PASSWORD           | Postgres database password                                        | `''`                                  |
+| POSTHOG_POSTGRES_HOST         | Postgres database host                                            | `'localhost'`                         |
+| POSTHOG_POSTGRES_PORT         | Postgres database port                                            | `5432`                                |
+| POSTHOG_POSTGRES_SSL_MODE     | Postgres database SSL mode                                        | `'disable'`                           |
+| POSTHOG_POSTGRES_CLI_SSL_CA   | Absolute path to the CA certificate to use for Postgres           | `null`                                |
```

Comment:
I think it makes sense to follow what the main PostHog repo does here: https://github.com/PostHog/posthog/blob/master/posthog/settings.py#L383-L387 - that is, passing the certs as env vars instead of files.

The same variables are used in both places

Reply:
I've made these changes, let me know if they don't work for you?

Reply:
.. My bad, looks like these are supposed to be `PATH`s, and the database URL takes a path, but when inserted into as options into `node-pg` client, taken in as the actual cert strings. Will revert to reading the file here.